plugins/odpcache/README.md

```diff
@@ -0,0 +1,65 @@
+# ODP Cache
+Use a ODP Cache to cache user segments fetched from the ODP server.
+
+## Out of Box Cache Usage
+
+1. To use the in-memory `ODPCache`, update the `config.yaml` as shown below:
+```
+## configure optional ODP Cache
+odp:
+  segmentsCache:
+    default: "in-memory"
```

Comment:
It can be better if default "default" is clarified. If no  `segmentsCache` is defined, we will use the default in-memory with default size and timeout?